Scheduler/observing_blocks_maintel/BLOCK-T609.json

```diff
@@ -0,0 +1,55 @@
+{
+    "name": "Imaging",
+    "program": "BLOCK-T609_1",
+    "constraints": [],
+    "scripts": [
+        {
+            "name": "maintel/track_target.py",
+            "standard": true,
+            "parameters": {
+                "slew_icrs": {
+                  "ra": "$ra",
+                  "dec": "$dec"
+                },
+                "rot_type": "PhysicalSky",
+                "az_wrap_strategy": "NOUNWRAP",
+                "program": "$program",
+                "reason": "$observation_reason"
+            }
+        },
+        {
+            "name": "maintel/disable_aos_closed_loop.py",
+            "standard": true,
+            "parameters": {}
+        },
+        {
+            "name": "maintel/take_aos_sequence_lsstcam.py",
+            "standard": true,
+            "parameters": {
+                "exposure_time": 30.0,
+                "n_sequences": 1,
+                "program": "$program",
+                "reason": "$observation_reason",
+                "note": "standard_aos_sequence"
+            }
+        },
+        {
+            "name": "maintel/take_aos_sequence_lsstcam.py",
+            "standard": true,
+            "parameters": {
+                "exposure_time": 60.0,
+                "dz": 7500,
+                "n_sequences": 1,
+                "program": "$program",
+                "reason": "$observation_reason",
+                "note": "dalmatian_pattern_investigation"
+            }
+        },
+        {
+            "name": "maintel/enable_aos_closed_loop.py",
+            "standard": true,
+            "parameters": {}
```

Comment:
Recommend you add some parameter overrides here to match what we're currently doing with the SV survey initial alignment block:
used_dofs: 0-16, 30-34
truncation_index: 12